### PR TITLE
Update part 2 sveltekit ut

### PR DESCRIPTION
**Pull request by roberto-butti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,9 @@
-# create-svelte
+# Ultimate Tutorial Series for SvelteKit
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
+Welcome to the Ultimate Tutorial Series for SvelteKit - you are in the right place to learn all about building a full-blown, multilingual website using Storyblok & SvelteKit. 
 
-## Creating a project
+Please follow the step by step instructions available in our documentation. You can find all the different parts on this overview page and take it from here.
```
Comment (josefineschaefer) [inline README.md]: Let's add the link to the tutorial here: https://www.storyblok.com/tp/the-storyblok-sveltekit-ultimate-tutorial

Comment (roberto-butti) [reply README.md]: updated, i update also the update-part-3  Pull Request :)

File: src/routes/+layout.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+export async function load() {
+  storyblokInit({
+    accessToken: "TxjwDdrurDCNvQwkIyREVwtt",
+    use: [apiPlugin],
+    components: {
+      feature: Feature,
+      grid: Grid,
+      page: Page,
+      teaser: Teaser,
+    },
```
Comment (josefineschaefer) [inline src/routes/+layout.js:17]: Do we have to include the option for the US region here as well? (commented out, but to make people aware)

File: vite-nohttps.config.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { sveltekit } from '@sveltejs/kit/vite';
```
Comment (josefineschaefer) [inline vite-nohttps.config.js:10]: This is just for Stackblitz, right? Maybe we should add a comment at the top for people to know that if they clone this, working locally, they won't need it?

**Review by josefineschaefer — APPROVED** [review]
Thanks so much, looks really good - just added comments / questions for some minor things :)

File: src/routes/+layout.js (hunk 2 above)
Comment (roberto-butti) [reply src/routes/+layout.js:17]: added! thank you for highlight this.

File: vite-nohttps.config.js (hunk 3 above)
Comment (roberto-butti) [reply vite-nohttps.config.js:10]: added the comment on the file

**Pull request merged**